external-crates/move/crates/move-vm-types/src/values/values_impl.rs

```diff
@@ -2438,6 +2693,35 @@ impl Struct {
     }
 }
 
+/***************************************************************************************
+ *
+ * Variant Operations
+ *
+ *   Public APIs for Enums.
+ *
+ **************************************************************************************/
+impl Variant {
```

Comment:
Are these used anywhere yet?

Reply:
Yep! These are used by the main VM interpreter loop for packing/unpacking and tag checking.